sonarts-core/src/utils/nodes.ts

```diff
@@ -233,3 +233,7 @@ export function isVariableStatement(node: ts.Node): node is ts.VariableStatement
 export function isUnionOrIntersectionTypeNode(node: ts.Node): node is ts.UnionOrIntersectionTypeNode {
   return is(node, ts.SyntaxKind.UnionType, ts.SyntaxKind.IntersectionType);
 }
+
+export function isSignatureDeclaration(node: ts.Node): node is ts.SignatureDeclaration {
+  return (<any>node).parameters !== undefined;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type. ([typescript:S4204](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4204))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&pullRequest=643&issues=AWNpQljGU2gg8QXllzFS&open=AWNpQljGU2gg8QXllzFS)

[ISSUE_KEY]: AWNpQljGU2gg8QXllzFS